src/clr-addons/brand-avatar/brand-avatar.html

```diff
@@ -0,0 +1,142 @@
346694,24.7361355 383.672162,24.7361355 C403.002087,24.7361355 420.794177,31.2812359 435.020271,42.2643757 C420.620335,60.8271134 412.021828,84.1458529 412.021828,109.5 C412.021828,134.851908 420.620335,158.175126 435.020271,176.735624 C420.794177,187.716525 402.99763,194.259386 383.672162,194.259386 C364.346694,194.259386 346.550147,187.716525 332.324053,176.735624 M194.981958,176.735624 C209.377437,158.175126 217.978172,134.851908 217.978172,109.5 C217.978172,84.1458529 209.377437,60.8226351 194.981958,42.2643757 C209.205823,31.2812359 227.00237,24.7361355 246.327838,24.7361355 C265.655535,24.7361355 283.454311,31.2812359 297.678176,42.2643757 C283.280468,60.8226351 274.679733,84.1458529 274.679733,109.5 C274.679733,134.851908 283.280468,158.175126 297.678176,176.735624 C283.454311,187.716525 265.655535,194.259386 246.327838,194.259386 C227.00237,194.259386 209.205823,187.716525 194.981958,176.735624 M24.6209715,109.5 C24.6209715,62.6833872 62.3960095,24.7361355 108.990201,24.7361355 C128.317897,24.7361355 146.114444,31.2812359 160.33831,42.2643757 C145.940602,60.8226351 137.339866,84.1458529 137.339866,109.5 C137.339866,134.851908 145.940602,158.175126 160.336081,176.735624 C146.112216,187.716525 128.317897,194.259386 108.990201,194.259386 C62.3960095,194.259386 24.6209715,156.312134 24.6209715,109.5 M177.657905,158.712527 C167.789012,144.834138 161.963066,127.856734 161.963066,109.5 C161.963066,91.1455053 167.791241,74.161384 177.657905,60.2852337 C187.531256,74.161384 193.357201,91.1455053 193.357201,109.5 C193.357201,127.856734 187.531256,144.834138 177.657905,158.712527 M314.997771,158.712527 C305.126649,144.834138 299.300704,127.856734 299.300704,109.5 C299.300704,91.1455053 305.126649,74.1658623 314.997771,60.2897121 C324.871122,74.1658623 330.694839,91.1455053 330.694839,109.5 C330.694839,127.856734 324.871122,144.834138 314.997771,158.712527 M452.339866,158.712527 C442.473202,144.834138 436.642799,127.856734 436.642799,109.5 C436.642799,91.1455053 442.473202,74.1658623 452.339866,60.2897121 C462.208759,74.1658623 468.034705,91.1455053 468.034705,109.5 C468.034705,127.856734 462.208759,144.834138 452.339866,158.712527 M521.012028,0 C494.978031,0 471.088089,9.18508445 452.342095,24.4853482 C433.593873,9.18508445 409.70393,0 383.672162,0 C357.638165,0 333.748222,9.18508445 314.997771,24.4831091 C296.254006,9.18508445 272.364064,0 246.327838,0 C220.293841,0 196.403899,9.18508445 177.657905,24.4831091 C158.916369,9.18508445 135.021969,0 108.990201,0 C48.7961934,0 0,49.0244366 0,109.5 C0,169.975563 48.7961934,219 108.990201,219 C135.021969,219 158.916369,209.814916 177.657905,194.512413 C196.403899,209.814916 220.293841,219 246.327838,219 C272.364064,219 296.254006,209.814916 314.997771,194.512413 C333.748222,209.814916 357.638165,219 383.672162,219 C409.70393,219 433.593873,209.814916 452.342095,194.512413 C471.088089,209.814916 494.978031,219 521.012028,219 C581.206035,219 630,169.975563 630,109.5 C630,49.0244366 581.206035,0 521.012028,0"
```

Comment:
pls move those icons to our custom icons in "icons" folder and reference it with clr-icon component